companies/lijuan/bubble-tea.php

```diff
@@ -0,0 +1,12 @@
+<?php include 'header.php';?>
```

Comment:
Let's follow the same standard by copy and paste my code first. Otherwise, you need to implement all the features by yourself or at least implement all the HTML & CSS by yourself. Your product page should look exactly like mine. Or it will result 6 times workload for you.

Reply:
You can take a look at the header.php file, I just put all your code there, so I can reuse it.